### ISSUE TITLE
Wont compile against Android 22

### ISSUE THREAD

**Issue by rob-lane** [Telerik-Verified-Plugins/Stripe#16]
The plugin seems to reference some items that are not supported by Android 22 (5.1) such as 'NameValuePair'.  These items seems to all come from the 'org.apache' package with is not supported any longer.

```
...
location: class StripePlugin
/Users/infuse/dev/moneyshot_frontend/platforms/android/src/com/telerik/stripe/StripePlugin.java:114: error: cannot find symbol
    HttpParams params = new BasicHttpParams();
                            ^
  symbol:   class BasicHttpParams
  location: class StripePlugin
/Users/infuse/dev/moneyshot_frontend/platforms/android/src/com/telerik/stripe/StripePlugin.java:115: error: cannot find symbol
    for (NameValuePair pair : nameValuePairs) {
         ^
  symbol:   class NameValuePair
  location: class StripePlugin
/Users/infuse/dev/moneyshot_frontend/platforms/android/src/com/telerik/stripe/StripePlugin.java:138: error: cannot find symbol
            nameValuePairs.add(new BasicNameValuePair(root + "[" + key + "]", jsonObject.getString(key)));
                                   ^
  symbol:   class BasicNameValuePair
  location: class StripePlugin
/Users/infuse/dev/moneyshot_frontend/platforms/android/src/com/telerik/stripe/StripePlugin.java:140: error: cannot find symbol
            nameValuePairs.add(new BasicNameValuePair(key, jsonObject.getString(key)));
                                   ^
  symbol:   class BasicNameValuePair
  location: class StripePlugin
/Users/infuse/dev/moneyshot_frontend/platforms/android/src/com/telerik/stripe/StripePlugin.java:151: error: cannot find symbol
    List<NameValuePair> nameValuePairs = new ArrayList<NameValuePair>();
         ^
  symbol:   class NameValuePair
  location: class StripePlugin
/Users/infuse/dev/moneyshot_frontend/platforms/android/src/com/telerik/stripe/StripePlugin.java:151: error: cannot find symbol
    List<NameValuePair> nameValuePairs = new ArrayList<NameValuePair>();
                                                       ^
  symbol:   class NameValuePair
  location: class StripePlugin
/Users/infuse/dev/moneyshot_frontend/platforms/android/src/com/telerik/stripe/StripePlugin.java:153: error: cannot find symbol
    for (NameValuePair pair : nameValuePairs) {
         ^
  symbol:   class NameValuePair
  location: class StripePlugin
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /Users/infuse/dev/moneyshot_frontend/platforms/android/src/io/card/cordova/sdk/CardIOCordovaPlugin.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
37 errors
 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 5.583 secs```

**Comment by EddyVerbruggen** [comment]
Thanks for the report, I'll look into it!

**Comment by EddyVerbruggen** [comment]
Fixed in 1.0.8
